.github/workflows/coverity.yml

```diff
@@ -0,0 +1,27 @@
+name: coverity-scan
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Once every day at 00:00 UTC
+env:
+  CC: gcc
+  ARCH: x86_64
+  CONTAINER_NAMESPACE: ghcr.io/opendataplane/odp-docker-images
+  OS: ubuntu_20.04
+  COVERITY_EMAIL: matias.elo@nokia.com
+  COVERITY_PROJECT: odp
+
+jobs:
+  Coverity-analysis:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
```

Comment:
Not needed, since there is no build matrix.

Reply:
Ack, will remove.